### DISCUSSION TITLE
What does package.json#source do?

### DISCUSSION THREAD

**Discussion by o-az** [vercel/turborepo#2984] [Help] [answered]
https://github.com/vercel/turbo/blob/22773fc9c5f03adbd543d6de4f88b4ca0120056a/examples/with-docker/packages/logger/package.json#L5-L7

I couldn't find any explanation for the field in the docs or in the example itself.

I tried googling, only relevant results are all [parceljs](https://github.com/parcel-bundler/parcel) related, which says:

[Source: Specify the entry points for your source code which gets mapped to your targets, can be a string or an array.](https://parcel2-docs.vercel.app/configuration/package-json/)

In turbo's case does that mean I don't need the `main` field?

**Comment by anthonyshew** [comment, marked as answer]
You're correct; `source` is not in the Node.js spec. I'm updating the example to drop it here: https://github.com/vercel/turbo/pull/6759

Thank you!
